src/styles/Home.css

```diff
@@ -7,6 +7,7 @@
   vertical-align: middle;
   justify-content: center;
   color: white;
+  position: relative;
```

Comment:
Home.css: Good use of `position: relative;` to be uniform with the Contact page.